mattermost-plugin/server/boards/boards_service_api.go

```diff
@@ -81,4 +80,5 @@ func (bs *boardsServiceAPI) DuplicateBoard(boardID string, userID string,
 }
 
 // Ensure boardsServiceAPI implements product.BoardsService interface.
-var _ product.BoardsService = (*boardsServiceAPI)(nil)
+// Commented out this becuase we are not using boards as a product.
+// var _ product.BoardsService = (*boardsServiceAPI)(nil)
```

Comment:
If no longer used, should we just delete this whole file that is implementing `boardsServiceAPI`? Is this reference at all in the plugin? Seems that this was for the Mattermost server to use as part of MPA which is no longer needed

Reply:
Okay, I have Checked and seems like this file no longer has any references in the focalboard. Removed it.